bqskit/ir/circuit.py

```diff
@@ -2143,193 +2221,145 @@ def surround(
                 f'Expected a positive integer num_qudits, got {num_qudits}.',
             )
 
-        if bounding_region is not None:
-            bounding_region = CircuitRegion(bounding_region)
+        if filter is not None and not callable(filter):
+            raise TypeError(f'Expected callable filter, got {type(filter)}.')
 
-        point = self.normalize_point(point)
+        def default_scoring_fn(region: CircuitRegion) -> float:
+            return float(sum(op.num_qudits for op in self[region]))
```

Comment:
Maybe this isn't super important as it's just a default, but this means single- and multi- qudit operations are very similarly important.

Reply:
No, this is an important point. I thought going by the number of qudits in a gate would be enough, but your comment made me realize this needs more. For example, a region with two single-qubit gates is equivalent to one with a two-qubit gate. How do you think we should solve this? Multiply by a factor, e.g., x100?